bp_common/syn/Makefile.vcs

```diff
@@ -39,6 +46,9 @@ VCS_BUILD_OPTS += +lint=TFIPC-L
 VCS_BUILD_OPTS += +plusarg_save
 VCS_BUILD_OPTS += $(BP_EXTERNAL_DIR)/lib/libdromajo_cosim.a
 VCS_BUILD_OPTS += -top test_bp
+ifeq ($(SYNTH_P), 1)
```

Comment:
Does vivado have to rename the toplevel? 

VCS is okay with having two top levels?

Reply:
Apparently yes. glbl needs to be added along with other top level modules according to this https://forums.xilinx.com/t5/Simulation-and-Verification/VCS-error-regarding-the-glbl-cross-module-reference-resolution/td-p/735596

Reply:
Add that link in a comment (wtf vivado)